server/core/pom.xml

```diff
@@ -23,6 +23,22 @@
          <groupId>io.netty</groupId>
          <artifactId>netty-handler</artifactId>
       </dependency>
+      <!-- Workaround for ISPN-8843 -->
+      <dependency>
+         <groupId>org.conscrypt</groupId>
+         <artifactId>conscrypt-openjdk</artifactId>
+         <version>${version.netty-conscrypt-optional}</version>
+         <classifier>${os.detected.classifier}</classifier>
```

Comment:
Shouldn't this be in test scope?

Reply:
let me try it and see it Intellij is happy

Reply:
It doesn't really matter since this is an `<optional>true</optional>` dependency.

Reply:
it doesn't work